avm/res/machine-learning-services/workspace/main.bicep

```diff
@@ -547,6 +562,87 @@ type featureStoreSettingType = {
   onlineStoreConnectionName: string?
 }?
 
+@discriminator('type')
+type OutboundRuleType = FqdnOutboundRuleType | PrivateEndpointOutboundRule | ServiceTagOutboundRule
+
+type FqdnOutboundRuleType = {
+  @sys.description('Required. Type of a managed network Outbound Rule of a machine learning workspace. Only supported when \'isolationMode\' is \'AllowOnlyApprovedOutbound\'.')
+  type: 'FQDN'
+
+  @sys.description('Required. Fully Qualified Domain Name to allow for outbound traffic.')
+  destination: string
+
+  @sys.description('')
+  category: 'Dependency' | 'Recommended' | 'Required' | 'UserDefined'?
+}
+
+type PrivateEndpointOutboundRule = {
+  @sys.description('Required. Type of a managed network Outbound Rule of a machine learning workspace. Only supported when \'isolationMode\' is \'AllowOnlyApprovedOutbound\' or \'AllowInternetOutbound\'.')
+  type: 'PrivateEndpoint'
+
+  @sys.description('Required. Service Tag destination for a Service Tag Outbound Rule for the managed network of a machine learning workspace.')
+  destination: {
+    @sys.description('Required. The resource ID of the target resource for the private endpoint.')
+    serviceResourceId: string
+
+    @sys.description('Optional. Whether the private endpoint can be used by jobs running on Spark.')
+    sparkEnabled: bool?
+
+    @sys.description('Required. The sub resource to connect for the private endpoint.')
+    subresourceTarget: string
+  }
+
+  @sys.description('')
```

Comment:
Missing description. Something along the lines of 'Optional. ...'

Reply:
Added